FastSimulation/Calorimetry/interface/CalorimetryManager.h

```diff
@@ -186,5 +188,8 @@ class CalorimetryManager{
   bool useShowerLibrary;
   bool useCorrectionSL;
   FastHFShowerLibrary *theHFShowerLibrary;
+
+  KKCorrectionFactors* ecalCorrection;
```

Comment:
I would like to check if the correction should be applied. Now, I check if the pointer is NULL, but this is not possible with a non-pointer. Do you recommend an additional variable 'bool doEcalCorrection' in the CalorimetryManager class?